packages/chain-helpers/src/blockchain/Blockchain.ts

```diff
@@ -167,26 +173,71 @@ export async function submitSignedTx(
 
 export const dispatchTx = submitSignedTx
 
+const metadataHashes = new Map<string, HexString>()
+
+// Returns the Merkle root of the metadata as stored in the `ConfigService` cache. If not present, it computes it, stores it in the cache for future retrievals, and returns it.
```

Comment:
Comment is outdated

Reply:
https://github.com/KILTprotocol/sdk-js/pull/924/commits/f5174bd3a7958e778d330a6b7828eaedccd16d64